#!/usr/bin/python3
def raise_exception_msg(message=""):
    try:
        geek = "GeeksforGeeks"
        return geeksforgeeks
    except NameError:
        return "NameError occurred. Some variable isn't defined."
